<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>331</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Select serial port</string>
  </property>
  <widget class="QGroupBox" name="selectBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Serial Port</string>
   </property>
   <widget class="QComboBox" name="serialPortInfoListBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>271</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="DescriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ManufacturerLabel">
       <property name="text">
        <string>Manufacturer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SerialNumberLabel">
       <property name="text">
        <string>Serial number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LocationLabel">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="vidLabel">
       <property name="text">
        <string>Vendor ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pidLabel">
       <property name="text">
        <string>Product ID:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ParametersBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Select Parameters</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="BaudRateLabel">
           <property name="text">
            <string>BaudRate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baudRateBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="DatabitsLabel">
           <property name="text">
            <string>Data bits:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dataBitsBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="ParityLabel">
           <property name="text">
            <string>Parity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="parityBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="StopBitsLabel">
           <property name="text">
            <string>Stop bits:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="stopBitsBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="FlowControlLabel">
           <property name="text">
            <string>Flow control:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="flowControlBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="additionalOptionsBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>229</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Additional options</string>
   </property>
   <widget class="QCheckBox" name="localEchoCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Local echo</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>290</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>248</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
